recipe/doc/end_user_run_guide.md

```diff
@@ -0,0 +1,121 @@
+Therefore, in a CUDA 11 world the conda-forge and nvidia channels were difficult to use in the same environment without some care.
+
+### CUDA 12.0-12.4
+
+With the CUDA 12 release, NVIDIA contributed the new packaging structure to conda-forge, introducing the same set of packages that existed on the nvidia channel as a replacement for the old `cudatoolkit` package on conda-forge.This was done starting with CUDA 12.0 to indicate the breaking nature of these changes compared to the prior CUDA 11.x packaging in conda-forge.
+These packages became the standard mechanism for delivering CUDA conda packages.
+Due to the scale of the reorganization, the CUDA 12.0, 12.1, and 12.2 releases also involved numerous additional fixes to the packaging structure to better integrate them in the Conda ecosystem.
+Due to the number of such changes that were required and the focus on improving the quality of these installations, during this time period no corresponding updates were provided for packages on the nvidia channel.
+While the conda-forge and nvidia channel package lists were the same (i.e. the same packages existed in both places with the same core contents like libraries and headers), the nvidia channel did not include many of the incremental fixes made on conda-forge to improve things like symlinks, static library handling, proper package constraints, etc.
+As a result, nvidia and conda-forge CUDA packages remained incompatible from CUDA 12.0-12.4.
+
+### CUDA 12.5+
+
+With CUDA 12.5, the nvidia channel was fully aligned with conda-forge.
+Packages on both channels are identical, ensuring safe coexistence of the two channels within the same conda environment.
+
+Going forward, the packages on the two channels should be expected to remain compatible.
+However, due to its smaller ecosystem footprint the nvidia channel may be a bit more nimble 
+
+## FAQ
+
+### What if I see an error saying `__cuda` is too old?
+
+You will need to update your driver
+- [Linux instructions](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#driver-installation)
+- [Windows instructions](https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html#installing-cuda-development-tools)
+
+If conda has incorrectly identified the CUDA driver, you can override by setting the `CONDA_OVERRIDE_CUDA` environment variable.
+
+### Can I install CUDA conda packages in a CPU-only environment (such as free-tier CIs)?
+
+Yes! All of the CUDA packages can be installed in an environment without the presence of a physical GPU or CUDA driver.
+The inter-package dependency is established properly so that this use case is covered.
+If you want to test package installation assuming a certain driver version is installed, use the `CONDA_OVERRIDE_CUDA` environment variable mentioned above.
```

Comment:
An extra sentence might help elaborate on the point that this is trying to make. I think it's a really useful tool, so I don't want this to be lost on the reader.
```suggestion
If you want to test package installation assuming a certain driver version is installed, use the `CONDA_OVERRIDE_CUDA` environment variable mentioned above.
Even if the package requires CUDA to run, this allows the packaging and dependency resolution to be tested in a CPU-only environment.
```